servers_v7.json

```diff
@@ -1,4 +1,8 @@
 [
+   {
+    "name": "Anana|Origin",
+    "address": ["mdtleague.top"]
```

Comment:
it makes no sense to put your server at the very top

Reply:
I just casually added it. The read position is not considered.

Reply:
It doesn't matter where you put it.